crates/autopilot/src/arguments.rs

```diff
@@ -593,6 +599,20 @@ pub struct FeePoliciesConfig {
     /// Volume fee policies that will become effective at a future timestamp.
     #[clap(flatten)]
     pub upcoming_fee_policies: UpcomingFeePolicies,
+
+    /// Custom volume fees for token buckets.
+    /// Format: "factor:token1;token2;..." (e.g.,
+    /// "0:0xA0b86...;0x6B175...;0xdAC17...") Orders where BOTH tokens are
+    /// in the bucket will use the custom fee. Useful for
+    /// stablecoin-to-stablecoin trades or specific token pairs (2-token
+    /// buckets). Multiple buckets can be separated by commas.
+    #[clap(long, env, value_delimiter = ',')]
+    pub volume_fee_bucket_overrides: Vec<TokenBucketFeeOverride>,
```

Comment:
These arguments are currently duplicated in the autopilot and orderbook. You can add them to the `shared` arguments to keep them in sync.